src/vs/workbench/contrib/issue/browser/issueQuickAccess.ts

```diff
@@ -0,0 +1,150 @@
+		const productLabel = this.productService.nameLong === 'stable' ? localize("workbench.action.openIssueReporter", "Visual Studio Code") : localize("workbench.action.openIssueReporterInsiders", "Visual Studio Code: Insiders");
+		issuePicks.push({
+			label: productLabel,
+			ariaLabel: productLabel,
+			accept: () => this.commandService.executeCommand('workbench.action.openIssueReporter', { issueSource: IssueSource.VSCode })
+		});
+
+		issuePicks.push({ type: 'separator' });
+
+		const marketPlaceLabel = localize("workbench.action.openIssueReporter2", "Extension Marketplace");
+		issuePicks.push({
+			label: marketPlaceLabel,
+			ariaLabel: marketPlaceLabel,
+			accept: () => this.commandService.executeCommand('workbench.action.openIssueReporter', { issueSource: IssueSource.Marketplace })
+		});
+
+		issuePicks.push({ type: 'separator', label: localize('extensions', "Extensions: Custom Reporting") });
+
+		// creates menu from contributed
+		const menu = this.menuService.createMenu(MenuId.IssueReporter, this.contextKeyService);
+
+		// render menu and dispose
+		const actions = menu.getActions({ renderShortTitle: true }).flatMap(entry => entry[1]);
+
+		// create picks from contributed menu
+		actions.forEach(action => {
+			if ('source' in action.item && action.item.source) {
+				extensionIdSet.add(action.item.source.id);
+			}
+
+			const pick = this._createPick(filter, action);
+			if (pick) {
+				issuePicks.push(pick);
+			}
+		});
+
+
+		issuePicks.push({ type: 'separator', label: localize('otherExtensions', "Other Extensions") });
+
+
+		// create picks from extensions
+		this.extensionService.extensions.forEach(extension => {
+			if (!extension.isBuiltin) {
+				const pick = this._createPick(filter, undefined, extension);
+				const id = extension.identifier.value;
+				if (pick) {
+					if (extensionIdSet.has(id)) {
+						return;
+					}
+					else {
+						issuePicks.push(pick);
+					}
+				}
+				extensionIdSet.add(id);
+			}
+		});
+
+		return issuePicks;
+	}
+
+	private _createPick(filter: string, action?: MenuItemAction | SubmenuItemAction | undefined, extension?: IRelaxedExtensionDescription): IPickerQuickAccessItem | undefined {
+		if (action && 'source' in action.item && action.item.source) {
+			const label = action.item.source?.title;
+			const highlights = matchesFuzzy(filter, label, true);
+			if (highlights) {
+				return {
+					label,
+					highlights: { label: highlights },
+					buttons: [{
+						iconClass: ThemeIcon.asClassName(Codicon.info),
+						tooltip: localize('contributedIssuePage', "Open Extension Page")
+					}],
+					trigger: () => {
+						if ('source' in action.item && action.item.source) {
+							this.commandService.executeCommand('extension.open', action.item.source.id);
+						}
+						return TriggerAction.CLOSE_PICKER;
+					},
+					accept: (keyMod, event) => {
+						action.run();
+					}
+				};
+			}
+		} else if (extension && extension.displayName && extension.identifier.value) {
```

Comment:
```suggestion
		} else if (extension?.displayName) {
```
this is equivalent because `identifier.value` is always defined.